### PR TITLE
Nested value of type any in update_status fix

### PR DESCRIPTION
**Pull request by nitesh-idnani1**
Fixed nested value initialization of type any in update_status by adding existence check in fix_null_objects module.

### CODE REVIEW

File: papiea-sdk/typescript/__tests__/sdk_tests/provider_sdk.test.ts [hunk 1]
```diff
@@ -964,14 +984,82 @@ describe("Provider Sdk tests", () => {
                 }
             });
 
-            const res: any = await axios.post(`${sdk.entity_url}/${sdk.provider.prefix}/${sdk.provider.version}/${kind_name}/${metadata.uuid}/procedure/testProcedure`, { input: '2' });
+            const res: any = await axios.post(`${sdk.entity_url}/${sdk.provider.prefix}/${sdk.provider.version}/${kind_name}/${metadata.uuid}/procedure/testProcedure`, null);
```
Comment (shlomi-nx) [inline papiea-sdk/typescript/__tests__/sdk_tests/provider_sdk.test.ts:987]: I see this test as well as the next don't take any input parameters. I would keep at least one with input parameters (the schema can be in the source, no need for external file). But its good to test different paths.

**Review by shlomi-nx — COMMENTED** [review]
Looks good, minor comment

File: papiea-sdk/typescript/__tests__/sdk_tests/provider_sdk.test.ts (hunk 1 above)
Comment (nitesh-idnani1) [reply papiea-sdk/typescript/__tests__/sdk_tests/provider_sdk.test.ts]: Changed one test to take an input parameter

File: papiea-sdk/typescript/__tests__/sdk_tests/provider_sdk.test.ts [hunk 2]
```diff
@@ -928,18 +928,38 @@ describe("Provider Sdk tests", () => {
...
+                            type: 'string'
+                        },
+                        account_id: {
+                            type: 'string'
+                        }
+                    }
+                }
+            }
+        }
+    }
```
Comment (shlomi-nx) [inline papiea-sdk/typescript/__tests__/sdk_tests/provider_sdk.test.ts:951]: I wouldn't use schemas from bmaas. Please be aware that Papiea is open and everyone could read its contents, while bmaas is closed. Please come up with an example schema which does not leak proprietary information.

**Review by shlomi-nx — CHANGES_REQUESTED** [review]
Please see comment

File: papiea-sdk/typescript/__tests__/sdk_tests/provider_sdk.test.ts (hunk 2 above)
Comment (nitesh-idnani1) [reply papiea-sdk/typescript/__tests__/sdk_tests/provider_sdk.test.ts:951]: Ohh I'm sorry, I'll change this to a generic one. Thanks for pointing this out.

**Review by Emulebest — APPROVED** [review]

**Pull request merged**